Source/WebCore/page/NavigateEvent.cpp

```diff
@@ -52,13 +55,13 @@ NavigateEvent::NavigateEvent(const AtomString& type, const NavigateEvent::Init&
 
 Ref<NavigateEvent> NavigateEvent::create(const AtomString& type, const NavigateEvent::Init& init, AbortController* abortController)
 {
-    return adoptRef(*new NavigateEvent(type, init, abortController));
+    return adoptRef(*new NavigateEvent(type, init, EventIsTrusted::Yes, abortController));
 }
 
 Ref<NavigateEvent> NavigateEvent::create(const AtomString& type, const NavigateEvent::Init& init)
 {
     // FIXME: AbortController is required but JS bindings need to create it with one.
-    return adoptRef(*new NavigateEvent(type, init, nullptr));
+    return adoptRef(*new NavigateEvent(type, init, EventIsTrusted::No, nullptr));
```

Comment:
Why is one trusted and the other not?

Reply:
This one is called from JavaScript. The other is only exposed to WebCore.

Reply:
This is a separate change, but only the scroll tests hit it...